packages/block-library/src/post-featured-image/block.json

```diff
@@ -37,6 +37,18 @@
 		"spacing": {
 			"margin": true,
 			"padding": true
+		},
+		"__experimentalBorder": {
+			"color": true,
+			"radius": true,
+			"style": true,
+			"width": true,
+			"__experimentalDefaultControls": {
+				"color": false,
+				"radius": false,
+				"style": false,
+				"width": false
+			}
```

Comment:
If all the border controls are in fact to be optional, I think you can omit this part of the configuration.

That said, when the border support panel was first switched to the `ToolsPanel`, the general idea was that any block opting into more than the border radius support should have all its border support features as default controls. 

From memory, this was primarily to address UX issues around the split border color, width, and style controls. The fact this clutters the sidebar was also a driving force in refining the border controls and delaying opting into border support more widely.

Some background and context here can be found in https://github.com/WordPress/gutenberg/pull/34061 and https://github.com/WordPress/gutenberg/pull/36942.